src/encoded/static/components/item-pages/StructuralVariantSampleView/SvSampleTabBody.js

```diff
@@ -28,7 +28,13 @@ export const SvSampleTabBody = (props) => {
             <div className="row flex-column flex-lg-row">
                 <div className="inner-card-section col pb-2 pb-lg-0">
                     <div className="info-header-title">
-                        <h4>Manta - Caller Properties</h4>
+                        <h4>
+                            Structural Variant - Caller Properties (Manta)
+                            <a href="https://cgap-pipeline-main.readthedocs.io/en/latest/Pipelines/Downstream/SV_germline/Pages/SV_germline-step-part-1.html#manta" rel="noopener noreferrer" target="_blank"
```

Comment:
This link and the BIC-seq2 link below are different in nature, and I think this one should probably be adjusted to be similar to the BIC-seq2 one. As is, this link goes to a particular step in the SV pipeline, whereas the BIC-seq2 link goes to the overview of the pipeline, which feels more appropriate.
```suggestion
                            <a href="https://cgap-pipeline-main.readthedocs.io/en/latest/Pipelines/Downstream/SV_germline/overview-SV_germline.html" rel="noopener noreferrer" target="_blank"
```